src/test/java/com/flowingcode/vaadin/addons/markdown/MarkdownEditorDemo.java

```diff
@@ -49,7 +49,7 @@ public MarkdownEditorDemo() {
           mde.setDataColorMode(DataColorMode.DARK);
           break;
         case "Light":
-          mde.setDataColorMode(DataColorMode.LIGTH);
+          mde.setDataColorMode(DataColorMode.LIGHT);
```

Comment:
I didn't want to introduce a breaking change, so I decided to deprecate the enum value with the typo (and introduce the one with correct spelling), so that is why the typo still exist, I'm planning to eliminate it in a new major version in the future.